src/View/Helper/FormHelper.php

```diff
@@ -102,6 +109,11 @@ public function __construct(View $View, array $config = [])
 
         $this->_defaultWidgets = $this->_widgets + $this->_defaultWidgets;
 
+        $this->_controlMethod = 'input';
```

Comment:
But shouldn't it be the other way around: control() now by default, and if !method exists we fall back to the old input?